src/components/contextIcon/contextIcon.js

```diff
@@ -126,6 +127,8 @@ const ContextIcon = ({ symbol, ...props }) => {
       return <PficonNetworkRangeIcon {...props} />;
     case ContextIconVariant.openshift:
       return <OpenshiftIcon {...props} />;
+    case ContextIconVariant.acs:
+      return <OpenshiftIcon {...props} />;
```

Comment:
I wonder if there's a better icon for ACS

Reply:
we probably want the same icon we see when we install ACS from OperatorHub in OpenShift.

<img width="409" alt="image" src="https://github.com/quipucords/quipucords-ui/assets/5797328/9972a669-9909-4bd4-8bee-f2c0862eb8e4">

Reply:
yes, that would be ideal.

Reply:
@cdcabrera I see that on patternfly the icon for ACS is not available. https://www.patternfly.org/2022.05/guidelines/icons
Is there an easy way to import a product icon to patternfly or is it not the way it works ? Thanks!
https://docs.google.com/presentation/d/1M1DhoaO3Rnkv5IKWSSZnZvH0SOxsbVDHyDz0k2SgzTY/edit#slide=id.g21c2e5b05dd_0_0